.jshintrc

```diff
@@ -1,6 +1,4 @@
 {
-  "camelcase": true,
-  "curly": true,
```

Comment:
I'd tend to agree with you here. Hard rules which hurt readability (when they are supposed to actually help) are a bad thing. @pon what do you think?